src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+export default class App extends React.Component {
```

Comment:
- [ ] Kindly remove all `eslint-disable` comments in all your JavaScript files. Linters help us write clean and maintainable codes, therefore using `eslint-disable` comments defeats the purpose of using linters when creating an application. Kindly remove all of them. 👍

Reply:
It does harm to me. The project asks to write classes, and the linter prefers stateless functions to classes. So I invent a state, but the linter requires it to be used. Then I invent a use, and the linter requires one more line to destructure what is not structured at all.